content/en/docs/usage/kubectl-plugin.md

```diff
@@ -79,4 +81,22 @@ The default output will be printed to stdout in YAML format. One can use -o opti
 For example this will output `cert.yaml` in the latest API version:
 ```console
 kubectl cert-manager convert -f cert.yaml
+```
+
+### Create
+`kubectl cert-manager create` can be used to create cert-manager resources manually. Sub-commands are available
+to create different resources:
+#### certificaterequest
```

Comment:
Could we add that the certificate (and private key) itself will not be stored inside Kubernetes?